win32/devdoc/threadpool_win32.md

```diff
@@ -56,6 +56,24 @@ MOCKABLE_FUNCTION(, THANDLE(THREADPOOL), threadpool_create, EXECUTION_ENGINE_HAN
 
 **SRS_THREADPOOL_WIN32_01_025: [** `threadpool_create` shall obtain the PTP_POOL from the execution engine by calling `execution_engine_win32_get_threadpool`. **]**
 
+[Will be moved from threadpool_open.]
+**S_R_S_THREADPOOL_WIN32_01_011: [** Otherwise, `threadpool_create` shall switch the state to OPENING. **]**
```

Comment:
>OPENING [](http://example.com/codeflow?start=94&length=7)

There is no OPENING state any longer, right?

Reply:
Are there any state transition tests in libraries that use this APIs like Queue Processor

Reply:
Removed